### ISSUE TITLE
Diagnostics: name/description and normalization

### ISSUE THREAD

**Issue by crocha700** [pyqg/pyqg#90]
I know this may sound perfectionism for some. Current description of the diagnostics is misleading. These terms represent tendencies in the "spectral energy density". The spectrum has units of . The have units of (L^2 T^{-3}) per unit wavenumbe. For example, if the wanumber is in cycles per km, and energy in m^2/s^2, then this would be m^2 s^-3 / cpkm.

My main concern is:
1. The "flux terms"  are actually *divergence* of flux...To get the flux we must integrate in wavenumber.
2.  "APEgen" is actually represents the spectrum of the rate of potential energy generation.
3. "Total energy dissipation" should be the spectrum of mechanical energy dissipation through bottom drag.
etc.

Also, the implementation of the fft computes an [unnormalized](http://www.fftw.org/doc/The-1d-Discrete-Fourier-Transform-_0028DFT_0029.html#The-1d-Discrete-Fourier-Transform-_0028DFT_0029) transform. Note that because we're using pyfftw, this does not matter if we are just transforming back and forth. But,  to get the correct magnitudes for the diagnostics we need to normalize to normalize things by the size of the array... In particular for quadratic quantities, such as the energy spectral density, we need to normalize by  M^2, where M = Nx x Ny.

**Comment by MFJansen** [comment]
I agree that the "flux terms" are really the (spectrally) local tendencies due to the nonlinear transfers - that is really spectral flux convergences. That should probably be clarified.

APEgen and EKEdiss I believe are actually be the total rates not the spectra though.

The normalizations may need to be double checked. I believe I had them worked out correctly at some point, but things may have changed since we are using different ffts.
